osu.Game/Overlays/Toolbar/Toolbar.cs

```diff
@@ -53,7 +53,7 @@ protected override void UpdateAfterChildren()
             base.UpdateAfterChildren();
 
             // this only needed to be set for the initial LoadComplete/Update, so layout completes and gets buttons in a state they can correctly handle keyboard input for hotkeys.
-            AlwaysPresent = false;
+            AlwaysPresent = true;
```

Comment:
This change, combined with the comment immediately preceding it, makes no sense.

This is already set to `true` in the constructor. So instead of changing this to set `true` every update again, it would make more sense to *remove* both lines.

I assume you tried to add this more locally in `ToolbarRulesetSelector` and it didn't work?

Reply:
I think I misunderstood the problem, so would just getting rid of both of those lines would fix it. I normally try to not delete lines when contributing to a project, that isn’t mine especially if it’s my first contribution.

Reply:
I understand your suggestion, and Tbh I’m not sure where that mindset. So we should just get rid of the  UpdateAfterChildren? (As I write this I understand more fully)